File: week-03/day-3/07.py
# create a function that takes a list and returns a new list with all the elements doubled

my_list = [1, 2, 3]

def double_list (input):
    new_list = input * 2
    return new_list
print (double_list(my_list))


def doubled_elements (input):
    doubled_list = []
    for i in range(len(input)):
        doubled_list += [input[i] * 2]
    return uj
print (doubled_elements(my_list))

File: week-07/day-2/07.js
'use strict';

var numbers = [2, 5, 11, 29];

// create a function that takes an array of numbers and returns a boolean
// it should return true if all the elements are prime, false otherwise

function isPrime(inputArray) {
  return inputArray.every(function (e) {
    if (e < 2) {
      return false;
    }
    for (var i = 3; i < e; i++) {
      if (e % i === 0) {
        return false;
      }
    }
    return true;
  });
}

console.log(isPrime(numbers));
